content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Introducing the Istio Operator
+description: Introduction to Istio's new operator-based installation and control plane management feature.
+publishdate: 2019-11-12
+subtitle:
+attribution: Martin Ostrowski (Google), Frank Budinsky (IBM)
+keywords: [install,configuration,istioctl,operator]
+target_release: 1.4
+---
+
+Kubernetes [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) provide
+a pattern for encoding human operations knowledge in software and are a popular way to simplify
+the administration of software infrastructure components.
+Because Istio is complex to administer, it's a natural candidate for an operator.
+
+Up until now, [Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio.
+This approach worked well but had some disadvantages:
+
+1. Users need to install another tool.
+1. Supporting Istio's many small specializations cause Helm templates to become unwieldy.
+1. Helm configurations are difficult to validate using Istio's proto-based APIs.
+1. Upgrades sometimes require Istio specific hooks that have been difficult to implement and maintain with Helm.
+
+Starting with Istio 1.4, the [Helm installation](/docs/setup/install/helm/) approach has been deprecated
+in favor of a new [installation using {{< istioctl >}}](/docs/setup/install/istioctl/) approach.
+Upgrading from Istio 1.4 onward (that is, versions not initially installed with Helm)
+will also be done using a new [{{< istioctl >}} upgrade feature](/docs/setup/upgrade/istioctl-upgrade/).
+
+The new `istioctl` commands use a
+[Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+to configure the installation.
+The CRD is part of a new Istio operator implementation intended to simplify the common administrative tasks of
+installation, upgrade, and complex configuration changes for Istio.
+Validation and checking for installation and upgrade is tightly integrated with the tools to prevent
+common errors and simplify troubleshooting.
+
```

Comment:
I wonder if we could insert a counterpoint section to the helm disadvantages section here.

## Istio Operator benefits

This new CR approach has several benefits:
1. Users just install one tool, Istio Operator, and it handles everything else
1. Users only need to specify the fields they want to customize in their IstioControlPlane resource
1. The Kubernetes API server automatically validates the schema's usage
1. The Istio Operator is a natural place to allow potentially complex Istio upgrade logic and insulate the user from this complexity

Reply:
agreed, I think it's better to avoid talking negatively about helm. It has some issues but it's also been a good tool for us.

Reply:
I don't think a new section is warranted, but I do think it would be fine to change the current wording in the overview to list the reasons for switching to operator as benefits instead of helm disadvantages.